.github/storypoints.py

```diff
@@ -0,0 +1,148 @@
+#! /usr/bin/env python
```

Comment:
I did not change this file at all

Reply:
We should probably look again and see if there are existing API clients we can use — I don't remember know what frustrations led me to making my own thin clients. But seems fine to me to use these for the time being.

Reply:
Also, I'm interested in @thatbudakguy 's opinion on the value and effort of converting some or all of this to use javascript actions and leveraging the builtin GitHub client for the GitHub API steps.